portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,4 +46,26 @@ private String convertToJson(List<String> msgs) {
     String json = gson.toJson(msgs);
     return json;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String newComment = getComment(request);
+    if (newComment != null) {
+        comments.add(newComment);
+
+        Entity commentEntity = new Entity("comment");
+        commentEntity.setProperty("text", newComment);
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Honestly I'm not too sure which way is recommended. I followed the guide and its suggested implementation, but I agree that it may get expensive quickly.

Reply:
Sounds good.
I was hoping there was some guidance from the instructions.